integration_tests/src/main/python/json_test.py

```diff
@@ -599,7 +599,7 @@ def test_from_json_map_fallback():
 @allow_non_gpu(*non_utc_allow)
 def test_from_json_struct(schema):
     # note that column 'a' does not use leading zeroes due to https://github.com/NVIDIA/spark-rapids/issues/9588
-    json_string_gen = StringGen(r'{"a": [1-9]{0,5}, "b": "[A-Z]{0,5}", "c": 1\d\d\d}') \
+    json_string_gen = StringGen(r'{\'a\': [1-9]{0,5}, "b": \'[A-Z]{0,5}\', "c": 1\d\d\d}') \
```

Comment:
nit: could use fewer escape sequences with triple quotes
```suggestion
    json_string_gen = StringGen(r'''{'a': [1-9]{0,5}, "b": '[A-Z]{0,5}', "c": 1\d\d\d}''') \
```

Reply:
Makes sense. I will address this in a follow-up PR if that is ok since we need this to make it into the 24.02 release.